app/Nova/Actions/CreateUserFromAttendance.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Actions;
+
+use App\Jobs\CreateOrUpdateUserFromBuzzAPI;
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Collection;
+use Laravel\Nova\Actions\Action;
+use Laravel\Nova\Fields\ActionFields;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Http\Requests\NovaRequest;
+
+class CreateUserFromAttendance extends Action
+{
+    use InteractsWithQueue;
+    use Queueable;
+
+    /**
+     * Indicates if this action is only available on the resource detail view.
+     *
+     * @var bool
+     */
+    public $onlyOnDetail = true;
+
+    /**
+     * The text to be used for the action's confirm button.
+     *
+     * @var string
+     */
+    public $confirmButtonText = 'Create User';
+
+    /**
+     * Get the fields available on the action.
+     *
+     * @return array<int, \Laravel\Nova\Fields\Field>
+     */
+    public function fields(NovaRequest $request): array
+    {
+        return [
+            Text::make('Reason for Creation')
+                ->maxlength(255)
+                ->enforceMaxLength()
+                ->default('Created From Attendance Record'),
```

Comment:
I don't think this needs to be user input - you can just use the same reason as `AttendanceObserver`, or a different hardcoded reason if you'd like. https://github.com/RoboJackets/apiary/blob/2f51275227a613377efa0aeedf4d7ffe3b3d653b/app/Observers/AttendanceObserver.php#L19-L23
The creation reason isn't used for anything in particular, mostly just akin to an audit trail.